### PR TITLE
Exposes in API the Repo entity's Size and Isempty properties


### PR DESCRIPTION
**Pull request by guyzmo**
cf gogits/gogs#4484

### CODE REVIEW

File: repo.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ type Repository struct {
 	Description   string      `json:"description"`
 	Private       bool        `json:"private"`
 	Fork          bool        `json:"fork"`
+	Empty         bool        `json:"empty"`
+	Size          bool        `json:"size"`
```
Comment (bkcsoft) [inline repo.go]: Size is not a boolean :)

Comment (guyzmo) [inline repo.go]: fixed as you were writing that 😁

**Comment by guyzmo** [comment]
is there still an issue?

File: repo.go [hunk 2]
```diff
@@ -27,6 +27,9 @@ type Repository struct {
 	Description   string      `json:"description"`
 	Private       bool        `json:"private"`
 	Fork          bool        `json:"fork"`
+	Parent        *Repository `json:"parent"`
```
Comment (Unknwon) [inline repo.go]: Where does this parent come from?

**Comment by guyzmo** [comment]
is it ok?

FYI, I've had that same change accepted and merged into gitea, I did my best to make sure any changes they required from me has been mirrored in that branch, but I can have done some mistakes!

**Comment by Unknwon** [comment]
Yeah, I didn't notice the correlated PR in the Gogs mai repository. Should be able review and merge tonight.

**Comment by guyzmo** [comment]
brilliant 👍

**Comment by Unknwon** [comment]
Thank you!

**Pull request merged**